utils/pckgen

```diff
@@ -257,25 +258,34 @@ generate_rpm() {
     rpm_obtain_sources
     rpm_spec_prepare
 
+    . $SYS_INFO
+    if [[ "$ID_LIKE" =~ suse|sles ]] && [ -d /usr/src/packages ]; then
+        info "INFO: It appears that you are using SUSE Linux."\
```

Comment:
missing 'if' probably

Reply:
Everything till the semicolon is handled by the single if statement.

Reply:
I mean word 'if' in info.

Reply:
I think the info message is correct. :)